org.eclipse.xtext.ui.tests/src-longrunning/org/eclipse/xtext/ui/tests/refactoring/ProgressReportingTest.java

```diff
@@ -78,6 +80,11 @@ public void assertLogged(List<String> expectation) {
 				throw new ComparisonFailure("", expectedEventsDump, eventsDump);
 			}
 		}
+
+		public void assertLogged(String expectation) {
+			String eventsDump = IterableExtensions.join(events, "\n");
```

Comment:
Can be `String.join`

Reply:
@szarnekow definitely yes :)
I've updated the other occurrences https://github.com/eclipse/xtext/pull/2900/commits/2a47548b1391620ba8b75428eb42eb340cfe1b32